plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/BaseIcebergConnectorSmokeTest.java

```diff
@@ -160,6 +160,54 @@ public void testDeleteRowsConcurrently()
         }
     }
 
+    private long getLatestSnapshotId(String tableName)
```

Comment:
nit: Can we move `getLatestSnapshotId` after all the test methods ?

Reply:
I figured out that there is already a method that does the same thing `getMostRecentSnapshotId` and used that one instead.